nextflow/accession.nf

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Accession variant files and copy to public FTP.
+
+    Inputs:
+            --accession_props       properties files for accessioning
+            --project_accession     project accession
+            --instance_id           instance id to run accessioning
+            --public_dir            directory for files to be made public
+            --logs_dir              logs directory
+    """
+}
+
+params.accession_props = null
+params.project_accession = null
+params.instance_id = null
+params.public_dir = null
+params.logs_dir = null
+// executables
+params.executable = ["bgzip": "bgzip", "tabix": "tabix", "copy_to_ftp": "copy_to_ftp"]
+// java jars
+params.jar = ["accession_pipeline": "accession_pipeline"]
+// help
+params.help = null
+
+// Show help message
+if (params.help) exit 0, helpMessage()
```

Comment:
So even if we explicitly set params.help to null in the previous line, this will still work because we pass in the command line parameter?

Reply:
That's my understanding, and I just tried it to check that it works...  To be honest I find this a bit unintuitive as well, but I modelled it off @tcezard 's code [here](https://github.com/EBIvariation/eva-submission/blob/master/nextflow/prepare_brokering.nf#L22).

Reply:
Actually this is fine as it is.
`params.help` is initialise as `null` and then can be overridden by the command line. 
If it is not set on the command line then it is still `null` and interpreted as `false` otherwise it is set to `true` and show the help message.
The only weird bit is that we don't see where the command line args are being parsed and assigned to the params object. but that's the :mage_woman: magic of a DSL :mage:

Reply:
Makes sense, thanks for explaining!